src/main/java/racingcar/PrintView.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+public class PrintView {
+	private static final String RESULT_MESSAGE = "실행 결과";
+
+	public static void printEmptyLine() {
+		System.out.println();
+	}
+
+	public static void printResultMessage() {
+		System.out.println("\n" + RESULT_MESSAGE);
+	}
+
+	public static void printCarMoving(Car car) {
+		for(int i = 0; i < car.getPosition(); i++) {
+			System.out.print("-");
+		}
+
+		printEmptyLine();
+	}
```

Comment:
```suggestion
	public static void printResultMessage() {
		System.out.println("\n" + RESULT_MESSAGE);
	}

	public static void printCarMoving(Car car) {
		for(int i = 0; i < car.getPosition(); i++) {
			System.out.print("-");
		}

		printEmptyLine();
	}
	
        public static void printEmptyLine() {
		System.out.println();
	}
```
printEmptyLine를 printCarMoving 아래에 배치하면 좋겠네요
이래야 `printEmptyLine()` 메소드가 뭔지 확인할 때 코드를 다시 위로 올려보지 않아도 됩니다.

Reply:
옷 이건 생각치도 못했는데..! 섬세한 피드백 감사합니다 🙇